magma/Makefile

```diff
@@ -0,0 +1,21 @@
+include ../make-base/stubs.mk
+
+export MAGMA_PROJECTS=$(shell cat config.yml | grep ':project_path:' | sed -e 's/.*:project_path:\s*//g')
```

Comment:
This is picking up both the "development" and the "test" projects from the `config.yml` file, which was unexpected. Does that mean that we should only have "development" configs in `config.yml` and keep test configs elsewhere? Otherwise it seems like we'll get the test `labors` project in development and the `ipi` project in test?

Reply:
Or use something like [yq](https://github.com/mikefarah/yq) to pick just the ones that match the environment? Not sure what the best approach would be here to get tests and dev to work independently.

Reply:
I actually figured it was ok to have both in both environments anyways, which is why I didn't bother doing anything more complicated.
 
Ok, I'll go ahead and try parsing only the currnent environment's projects.